usr/lib/qubes-whonix/replace-ips

```diff
@@ -1,3 +1,5 @@
+
+
```

Comment:
I think these blank lines should probably be removed.

Reply:
Fixed